packages/website/src/content/docs/changelog/edge.mdx

```diff
@@ -22,6 +22,11 @@ import MarkdownAlert from "@/components/markdown/MarkdownAlert.astro";
 ### Added
 
 * Adds [authentik_atlas_mongodb_sso](/docs/main/reference/infrastructure-modules/direct/authentik/authentik_mongodb_atlas_sso) & [mongodb_atlas_identity_provider](/docs/main/reference/infrastructure-modules/direct/authentik/mongodb_atlas_identity_provider) module for setting up the Authentik Application and Provider for Atlas MongoDB.
```

Comment:
I recall explicitly adding a comment to make this /docs/edge?

Reply:
the comment is made in another pr that is still open. it was uncaught on the prior pr.